### PR TITLE
Publish 3.4.4 to docker registry

### PR DESCRIPTION
**Pull request by dongjoon-hyun**
### What changes were proposed in this pull request?

Publish 3.4.4 to docker registry

### Why are the changes needed?

To provide the docker image.

### Does this PR introduce _any_ user-facing change?

No.

### How was this patch tested?

Pass the CIs.

### CODE REVIEW

**Comment by dongjoon-hyun** [comment]
cc @yaooqinn and @viirya

File: 3.4.4/scala2.12-java11-ubuntu/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    \
+    tar -xf spark.tgz --strip-components=1; \
+    chown -R spark:spark .; \
+    mv jars /opt/spark/; \
+    mv RELEASE /opt/spark/; \
+    mv bin /opt/spark/; \
+    mv sbin /opt/spark/; \
+    mv kubernetes/dockerfiles/spark/decom.sh /opt/; \
+    mv examples /opt/spark/; \
+    ln -s "$(basename $(ls /opt/spark/examples/jars/spark-examples_*.jar))" /opt/spark/examples/jars/spark-examples.jar; \
```
Comment (yaooqinn) [inline 3.4.4/scala2.12-java11-ubuntu/Dockerfile:63]: redundant ls

Comment (dongjoon-hyun) [reply 3.4.4/scala2.12-java11-ubuntu/Dockerfile:63]: Got it. This is generated by template.

It seems that the template still has the original form, @yaooqinn .

**Review by viirya — APPROVED** [review]

**Comment by dongjoon-hyun** [comment]
Thank you, @yaooqinn and @viirya .
Merged to master.

**Comment by yaooqinn** [comment]
FYI, official docker is ready  https://github.com/docker-library/official-images/pull/17826

**Comment by dongjoon-hyun** [comment]
Thank you for taking care of it, @yaooqinn !

**Pull request merged**